Automattic/sensei
# Add base data port model class and foundation for questions model

### Changes proposed in this Pull Request

* Adds base class for data port model.
* This method will handle defining the general structure of the data (both imported and exported).
* This also handles sanitation from the raw CSV data.
* I've added a WP CLI command to aid with development and speed up large imports in the future.
`wp sensei-import [--questions=path-to-file][ --courses=path-to-file][ --lessons=path-to-file]`

### Testing instructions

* Observe tests pass. (Note: `feature/importer` has a failing/unrelated JS build)
